lib/Sema/PreCheckExpr.cpp

```diff
@@ -238,6 +238,32 @@ static bool diagnoseRangeOperatorMisspell(DiagnosticEngine &Diags,
   return false;
 }
 
+static bool diagnoseUnexistingOperators(DiagnosticEngine &Diags,
```

Comment:
```suggestion
static bool diagnoseUnexistingOperators(DiagnosticEngine &Diags,
```

Same comment here. Also, since this method only diagnoses "**", perhaps that could be reflected in the name; it diagnoses no other operators.

Reply:
Thanks :) I modified it to `diagnoseNonexistentPowerOperator`